hypertrace-gradle-docker-java-application-plugin/src/main/resources/application-start-script.template.sh

```diff
@@ -23,4 +23,10 @@
 
 set -e
 
-exec java ${defaultJvmOpts.substring(1, defaultJvmOpts.length()-1)} \$JAVA_OPTS -classpath '/app/resources:/app/classes:/app/localLibs/*:/app/orgLibs/*:/app/externalLibs/*' ${mainClassName} \$@
+FIPS_CLASSPATH=""
+if [ "\$FIPS_ENABLED" = "true" ]; then
+  FIPS_CLASSPATH="/usr/share/java/bc-fips/*"
```

Comment:
Let's append to the classpath in here. We don't want the false case leaving this empty and starting the classpath with a `:`.